src/cli/Cargo.toml

```diff
@@ -49,4 +49,9 @@ flv-types = { path ="../types", version = "0.2.0" }
 flv-spu = { path = "../spu-server", optional = true }
 flv-sc-k8 = { path = "../sc-k8", optional = true }
 rpassword = "5.0.0"
+async-std = "1.6.3"
```

Comment:
unneeded since ```flv-future-aio``` re-exports async-std

Reply:
Hmm, I thought I had gotten rid of that. Give me just one minute and I'll fix it

Reply:
I'm running into a bit of a hiccup with this because some of the functions like `fs::write` require `path::Path`s as parameters, and while `async_std::fs` was re-exported from `flv-future-aio`, `async_std::path` was not.
It seems like I either need to keep the extra `async_std` dependency, or push an update to `flv-future-aio` with an `async_std::path` glob re-export`, or fall back to not using async io for this particular use case.

Reply:
For simple and non performance fs, you don't have to make async.